Framework/src/InformationService.h

```diff
@@ -38,7 +38,7 @@ namespace pt = boost::property_tree;
 /// See runInformationService.cxx for the steering code.
 ///
 /// Example usage :
-///      qcInfoService -c /absolute/path/to/InformationService.json -n information_service \\
+///      qcInfoService -c /absolute/path/to/InformationService.json -n information_service
```

Comment:
I couldn't think of another way to get rid of that warning, any ideas?

Reply:
ANy ways, this is going to disappear as it is replaced by the ServiceDiscovery of Adam and George.